File: Programmers/Lv0/조건에맞게수열변환하기1/solution.py
def condition(num: int):
    if num >= 50 and num % 2 == 0:
        return num // 2
    elif num < 50 and num % 2 != 0:
        return num * 2
    return num


def solution(arr):
    return [condition(n) for n in arr]

File: Programmers/Lv0/조건에맞게수열변환하기1/Solution.java
package Lv0.조건에맞게수열변환하기1;

import java.util.*;


class Solution {
    private int condition(int num) {
        if (num >= 50 && num % 2 == 0) return num / 2;
        else if (num < 50 && num % 2 != 0) return num * 2;
        return num;
    }

    public int[] solution(int[] arr) {
        return Arrays.stream(arr)
                .map(v -> condition(v))
                .toArray();
    }
}